SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+We take security issues in our project seriously and appreciate the efforts of the community in improving the security of our project.
+
+## Supported versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
```

Comment:
Do we actually support 2.1?

I don't think we're releasing hotfixes — actually, I don't even know if it's possible to release a patch to 2.1.x with the App Store?

Let's just include the most recent version here.

Reply:
Good point, adjusted to just note 2.7.